packages/mdc-select/mdc-select.scss

```diff
@@ -69,6 +68,23 @@
     background-position: left 10px center;
   }
 
+  &__dropdown-icon {
+    @include mdc-select-dd-arrow-svg-bg_($mdc-select-dropdown-color, $mdc-select-dropdown-opacity);
+    @include mdc-rtl-reflexive(left, auto, right, 8px);
+
+    position: absolute;
+    bottom: 16px;
```

Comment:
This seems like it sits higher vs. text in the default variant than in the outlined variant. Is there supposed to be a different `bottom` style for outlined?

Reply:
No, this is intended based on the design.